### DISCUSSION TITLE
How to use tables while creating a repository?

### DISCUSSION THREAD

**Discussion by anilneti** [community/community#49216] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello All,

I am creating a repo which has multiple sections in it. I would like to create a hyperlink on the home page so as to refer each section. Is this possible?

**Comment by jge162** [comment]
Yes easy to accomplish. Create a .md file and then use markdown syntax to link to each hyperlink as you need.
